plugins/engines/ospray/ispc/render/AdvancedSimulationRenderer.ispc

```diff
@@ -21,46 +21,35 @@
  */
 
 // Brayns
-#include <plugins/engines/ospray/ispc/render/utils/AbstractRenderer.ih>
+#include <plugins/engines/ospray/ispc/render/utils/SimulationRenderer.ih>
 
-// needs to be the same in MorphologyLoader.cpp
-uniform const uniform float OFFSET_MAGIC = 1e6;
-
-struct SimulationRenderer
+struct AdvancedSimulationRenderer
 {
-    AbstractRenderer abstract;
+    SimulationRenderer abstract;
```

Comment:
Is there a better name than abstract, like simulation_renderer or something?

Reply:
Dito